android-todo-list/architecture-samples-main/app/src/main/java/com/statsig/todoapp/data/ModelMappingExt.kt

```diff
@@ -90,3 +102,9 @@ fun NetworkTask.toExternal() = toLocal().toExternal()
 
 @JvmName("networkToExternal")
 fun List<NetworkTask>.toExternal() = map(NetworkTask::toExternal)
+
+fun String.getCurrentDateTime(): String {
```

Comment:
Does this need to live on String? Can it not just be a static function

Reply:
Will change it.